### ISSUE TITLE
Some accessibility suggestions

### ISSUE THREAD

**Issue by pattonwebz** [devodii/sidebars#1]
Hey, I just seen this repo mentioned on your linkedin and figured I would open the first repo issue with some accessibility suggestions :)

The `<nav>` element should have a name attribute to so it is easy to reach and can be uniquely identified.

The `<button>` should have some text content in it (even hidden as screen reader only text) or an `aria-label`. The svg inside the button should have `aria-hidden="true"`.

The <button> also should have `aria-expanded` with false when closed and true when open and an `aria-controls` pointing to the element that it opens.
